src/openrct2-ui/windows/Finances.cpp

```diff
@@ -879,6 +879,7 @@ static void window_finances_financial_graph_paint(rct_window* w, rct_drawpixelin
     {
         money32 axisValue = axisBase << yAxisScale;
         gfx_draw_string_right(dpi, STR_FINANCES_FINANCIAL_GRAPH_CASH_VALUE, &axisValue, COLOUR_BLACK, x + 70, y);
+        gfx_draw_line(dpi, x + 70, y + 5, graphLeft + 482, y + 5, w->colours[0]);
```

Comment:
To get to what I proposed in my last screenshot:

```suggestion
        gfx_fill_rect_inset(dpi, x + 70, y + 5, graphLeft + 482, y + 5, w->colours[2], INSET_RECT_FLAG_BORDER_INSET);
```


![Screenshot_20200301_161632](https://user-images.githubusercontent.com/604665/75628322-08fbba80-5bd8-11ea-9daf-fa9f345baa31.png)